### PR TITLE
Update to sklearn 0.20.0

### PR DESCRIPTION
**Pull request by john-sandall**
Closes https://github.com/manuel-calzolari/sklearn-genetic/issues/11, see discussion there.

### CODE REVIEW

**Comment by john-sandall** [comment]
I've tested this on Python 3.7 using the example given on the project README

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='sklearn-genetic',
-    version='0.2',
```
Comment (manuel-calzolari) [inline setup.py:13]: Let's keep the version unchanged at the moment.

File: setup.py [hunk 2]
```diff
@@ -33,6 +33,6 @@
         'Programming Language :: Python :: 3.7',
     ],
     packages=find_packages(),
-    python_requires='>=2.7',
```
Comment (manuel-calzolari) [inline setup.py:36]: Let's keep `python_requires` unchanged at the moment.

Comment (john-sandall) [reply setup.py:36]: Currently (in this PR) this library isn't compatible with 3.8, wouldn't you want to have that explicitly stated so tools like pipenv/pip-tools/conda can use this information to negotiate compatibility and dependency resolution? If your plan is to deploy a new version to PyPI incorporating changes from both this and #14 then disregard this message.

Comment (manuel-calzolari) [reply setup.py:36]: Yes, I'm going to also incorporate other changes, then I'll release a new version.

**Comment by manuel-calzolari** [comment]
Closed with commit c029c0e. Thank you very much for your effort!

**Pull request closed without merging**